README.md

```diff
@@ -600,6 +603,15 @@ __AWESOME__ apps counter: __140__ 🎉
 - [x] [GitHub](https://github.com/AndreAle94/moneywallet)
 - [ ] Official page
 
+## Firewall
```

Comment:
Sorry for asking, but I need to know something. On Android what is the difference between AdBlocking software and Firewall software on Android? I know AdBlocking tries to eliminate sketchy websites in comparison to Firewall software which is trying to block and allow some ports. However I want to make sure.

Reply:
You mostly right, the point is that firewall can block certain apps from accessing the different kinds of traffic. Say, if u have a preinstalled system app like weather interface that automatically sends some data if u connect through cellular and u don't have any builtin option to stop/restrict/remove it, then u can just block all cellular traffic access for this app but leave it a wifi access and use app normally when u need. Ofc you can as well setup cetain ports scripts, or even custom redirect to vpn/tor. Say, if you want your browser to always route traffic through tor but don't wanna enable orbot everytime, just set a chain rule to firewall. Also most adblock tools just remove elements programmatically from app's but not preventing the system load nor traffic passage to third-party servers.